pkg/pipeline/source/web.go

```diff
@@ -302,7 +302,7 @@ func (s *WebSource) navigate(chromeCtx context.Context, chromeCancel context.Can
 		switch ev := ev.(type) {
 		case *runtime.EventConsoleAPICalled:
 			if s.chromeLog != nil {
-				if b, err := ev.MarshalJSON(); err == nil {
+				if b, err := json.Marshal(ev); err == nil {
```

Comment:
@frostbyte73 @biglittlebigben is this ok?

Reply:
Should be. We won't know until we test though.